shell/shared/grain.js

```diff
@@ -1314,8 +1314,12 @@ if (Meteor.isClient) {
             // Store apitoken id1 and template in session storage in the offer
             // template namespace under key id2.
             var key = "offerTemplate" + id2;
+            var grain = getActiveGrain(globalGrains.get());
+            var grainTitle = grain.title();
+            var slug = grainTitle.toLowerCase().trim().replace(/[^\w ]+/g,'').replace(/ +/g,'-');
```

Comment:
Nitpick: It looks like a '-' in the title will actually get removed even though the final output can contain new '-'s, which seems odd. Maybe instead it makes sense to replace any span of non-\w characters with a single '-'.